### ISSUE TITLE
Config format standardization

### ISSUE THREAD

**Issue by lladdy** [Cryptyc/Sc2LadderServer#71]
Currently we have multiple configs that use differing formats.
I suggest we standardize any configs to all use the same format to allow the use of the same tools/libraries for all configs.
Json makes sense to me at this point in time.

I'm happy to take on this task - just want to make sure people are in support of it if I do.

**Comment by alkurbatov** [comment]
So, which format is preferred?

**Comment by lladdy** [comment]
Probably Json if that's what you mean: The change would be minimal because we already have rapidJson integrated.

**Comment by alkurbatov** [comment]
The only issue with json is that it is not detected by default on Windows, see:
https://github.com/davechurchill/commandcenter/issues/18

It could be also .ini (might be recognized on Windows by default) or .yaml which is very popular now.

**Comment by lladdy** [comment]
> The only issue with json is that it is not detected by default on Windows, see:
> davechurchill/commandcenter#18

That's a fair enough reason.

> It could be also .ini (might be recognized on Windows by default) or .yaml which is very popular now.

.ini is recognized on windows. Neither yml nor yaml are.
I would be eager to consider YAML as it seems a suitable candidate for configs in general, but my intention here is to merely standardize the formats first in order to simplify working with the configs (one file format, one parser, etc).
Maybe this is too many changes and we should first decide the final format?

If we moved the LadderManagers.conf file to json, my vote would be for maintaining the json extension for consistency's sake.
However, if we did move to a txt extension (or another recognized by Windows), then we should probably do the same for LadderBots.json as well.
